deployment/legacy-migration/maintenance-worker/migration/neo4j/users/users.cql

```diff
@@ -94,12 +94,11 @@
       }
     }
 */
-
 CALL apoc.load.json("file:${IMPORT_CHUNK_PATH_CQL_FILE}") YIELD value as user
 MERGE(u:User {id: user._id["$oid"]})
 ON CREATE SET
 u.name       = user.name,
-u.slug       = user.slug,
+u.slug       = COALESCE(user.slug, apoc.text.random(20, "[A-Za-z]")),
```

Comment:
from the docs `coalesce() returns the first non-null value in the given list of expressions.`
since `apoc.text.random()` will never return null, this is a safer way to import the data! 
Nice @roschaefer

Reply:
one thing I didn't notice though @roschaefer is that you are using `A-Z`, shouldn't we avoid using slug with capital letters?
I already merged it, but we could refactor, or you could explain why it's fine...

Reply:
@mattwr18 oh, yes you're right. We could also prefix the slug with `unknown-`... so these users show up at the end of the list.